website/content/en/pre-docs/tasks/running-pods.md

```diff
@@ -0,0 +1,222 @@
+{{% /alert %}}
+
+## Resource requests (`resources`)
+
+Within a Pod spec, you can both make requests and set limits on resources a pod needs, such as CPU and memory.
+For example:
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp
+spec:
+  containers:
+  - name: app
+    image: myimage
+    resources:
+      requests:
+        memory: "128Mi"
+        cpu: "500m"
+      limits:
+        memory: "256Mi"
+        cpu: "1000m"
+```
+In this example, the container is requesting 128MiB of memory and .5 CPU.
+Its limits are set to 256MiB of memory and 1 CPU.
+
+
+See [Managing Resources for Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) for details on resource types supported by Kubernetes, [Specify a memory request and a memory limit](https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/#specify-a-memory-request-and-a-memory-limit) for examples of memory requests, and [Specifying Values to Control AWS Provisioning](/docs/cloud-providers/aws/aws-spec-fields) for a list of supported resources.
+
+## Disruption budget (`PodDisruptionBudget`)
+
+Disruption budgets let you specify how much of a Deployment, ReplicationController, ReplicaSet, and StatefulSet must be protected from disruptions when pod eviction requests are made.
+This feature can be used to strike a balance between protecting the application's availability while still allowing a cluster operator to manage the cluster.
+For example:
+
+```
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: myapp-pdb
+spec:
+  minAvailable: 4
+  selector:
+    matchLabels:
+      app: myapp
+```
+In this example, the `myapp` pod is required to have at least 4 pods available after eviction.
+Instead of setting a number, you could set a percentage.
+So in this case if there were 10 pods, to get the same `minAvailable` you would set it to 40%.
+
+See [Specifying a Disruption Budget for your Application](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) for details.
+
+## Node selection (`nodeSelector` and `nodeName`)
+
+With `nodeSelector` you can ask for a node that matches selected key-value pairs.
+This can include well-known labels or custom labels you create yourself.
+
+While `nodeSelector` is like node affinity, it doesn't have the same "and/or" matchExpressions that affinity has.
+So all key-value pairs must match if you use `nodeSelector`.
+Also, `nodeSelector` can do only do inclusions, while `affinity can do inclusions and exclusions (`In` and `NotIn`).
+Here is an example:
+
+```
+nodeSelector:
+  topology.kubernetes.io/zone: us-west-2a
+  node.k8.aws/capacity-type: spot
+  jones.dev: john
+```
+This example features a well-known label (`topology.kubernetes.io/zone`), a label that is specific to the AWS cloud provider (`node.k8.aws/capacity-type`), and a custom label (`jonees.dev`).
```

Comment:
We removed custom label generation. Custom labels are used to select provisioners. If a pod declares a custom label, it will only be scheduled to a provisioner that has defined the same label.
```suggestion
This example features a well-known label (`topology.kubernetes.io/zone`), a label that is well-known to Karpenter (`karpenter.sh/capacity-type`).
```